test/regressions/webpack.config.js

```diff
@@ -42,4 +42,14 @@ module.exports = {
       },
     ]),
   },
+  resolve: {
+    ...webpackBaseConfig.resolve,
+    alias: {
+      ...webpackBaseConfig.resolve.alias,
+      docs: false, // Disable this alias as it creates a circual resolution loop with the docsx alias
```

Comment:
Since we are not supposed to import from the docs-infra or mono-repo docs in the visual regression tests, this works as well. It's cleaner than the first attempt:

```diff
-      docs: path.resolve(__dirname, './node_modules/@mui/monorepo/docs'),
+      docs: [
+        path.resolve(__dirname, './node_modules/@mui/monorepo/docs'),
+        path.resolve(__dirname, './docs'),
+      ],
```

Reply:
Actually, I might not have needed to fix this, I see that it's already all broken on HEAD, e.g. https://app.circleci.com/pipelines/github/mui/mui-x/25566/workflows/5b4b75f8-3d75-4d0f-8c6e-90812dd0563a/jobs/147039

<img width="829" alt="Screenshot 2022-10-09 at 02 35 11" src="https://user-images.githubusercontent.com/3165635/194732261-3672b583-9d5e-4b43-a08a-7ee7824884b0.png">